<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ACW-Lap-031" failures="2" tests="4" name="Errorvalidation" time="70.762" errors="0" timestamp="2024-06-27T01:17:59 IST">
  <testcase classname="framework.Testcase.Error_Validation_Test" name="logininvalid_Credencial" time="1.449">
    <skipped/>
  </testcase> <!-- logininvalid_Credencial -->
  <testcase classname="framework.Testcase.Error_Validation_Test" name="logininvalid_Credencial" time="1.112">
    <failure type="java.lang.AssertionError" message="expected [Incorrect email password.] but found [Incorrect email or password.]">
      <![CDATA[java.lang.AssertionError: expected [Incorrect email password.] but found [Incorrect email or password.]
at framework.Testcase.Error_Validation_Test.logininvalid_Credencial(Error_Validation_Test.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- logininvalid_Credencial -->
  <testcase classname="framework.Testcase.Error_Validation_Test" name="productErrorValidation" time="24.544">
    <skipped/>
  </testcase> <!-- productErrorValidation -->
  <testcase classname="framework.Testcase.Error_Validation_Test" name="productErrorValidation" time="14.825">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at framework.Testcase.Error_Validation_Test.productErrorValidation(Error_Validation_Test.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- productErrorValidation -->
</testsuite> <!-- Errorvalidation -->
